src/admin/components/SearchForm.scss

```diff
@@ -0,0 +1,28 @@
+.AdminSearchForm-form {
+  display: flex;
+
+  .AdminSearchForm-query {
+    background: url('~amo/img/icons/magnifying-glass.svg') 18px 50% no-repeat;
+    background-size: 26px;
```

Comment:
One thing about the old svg removed in this branch was that setting the size in the svg meant that background-size wasn't required.